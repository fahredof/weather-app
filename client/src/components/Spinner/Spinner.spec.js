import React from 'react';
import Spinner from './Spinner';

describe(`Spinner`, () => {
    const spinner = shallow(<Spinner/>);

    it(`Содержит класс: loader`, () => {
        expect(spinner.hasClass(`loader`)).toBeTruthy();
    });

    it(`отображается корректно`, () => {
        expect(shallowToJson(spinner)).toMatchSnapshot();
    });

});