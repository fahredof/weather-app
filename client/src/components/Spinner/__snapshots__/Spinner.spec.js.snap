// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Компонент содержит необходимые классы отображается корректно 1`] = `
<div
  className="loader"
/>
`;

exports[`Spinner отображается корректно 1`] = `
<div
  className="loader"
/>
`;
